src/web/utils.ts

```diff
@@ -54,3 +54,177 @@ export function calculateViewScale(view: HTMLElement) {
 
   return resultScales;
 }
+
+export function tryExtractStylusData(
+  event: PointerEvent
+): StylusData | undefined {
+  const pointerType = PointerTypeMapping.get(event.pointerType);
+
+  if (pointerType !== PointerType.STYLUS) {
+    return;
+  }
+
+  if (event.tiltX === 0 && event.tiltY === 0) {
+    // If we are in this branch, it means that either tilt properties are not supported and we have to calculate them from altitude and azimuth angles,
+    // or stylus is perpendicular to the screen and we can use altitude / azimuth instead of tilt
+
+    // If azimuth and altitude are undefined in this branch, it means that we are either perpendicular to the screen,
+    // or that none of the position sets is supported. In that case, we can treat stylus as perpendicular
+    //
+    // @ts-ignore This property exists (https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent#instance_properties)
+    if (event.azimuthAngle === undefined || event.altitudeAngle === undefined) {
```

Comment:
You can extract those to variables instead of `@ts-ignoring` multiple times

Reply:
Good idea. Done in [9fa4003](https://github.com/software-mansion/react-native-gesture-handler/pull/3107/commits/9fa4003dae72bdad1de3d9323fe6dcac43562963)